tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+"""Test various helper functions."""
+import numpy as np
+
+from pyprep.utils import _mat_quantile, _mat_iqr
+
+
+def test_mat_quantile_iqr():
+    """Test MATLAB-compatible quantile and IQR functions.
+
+    MATLAB code used to generate the comparison results:
+
+    .. code-block:: matlab
+
+       % Generate test data
+       rng(435656);
+       tst = rand(100, 3);
+
+       % Calculate IQR and 0.98 quantile for test data
+       quantile(tst, 0.98);
+       iqr(tst);
+
+    """
+    # Generate test data
+    np.random.seed(435656)
```

Comment:
so here the numpy seed corresponds to the matlab seed? (looking at the matlab codeblock `rng(435656);` in the docstring, line 15)

Reply:
Yep, that's correct: 435656 is the default MATLAB PREP seed.

Reply:
perfect, thanks!